wagtail/blocks/struct_block.py

```diff
@@ -11,41 +11,40 @@
 from wagtail.admin.staticfiles import versioned_static
 from wagtail.telepath import Adapter, register
 
-from .base import Block, BoundBlock, DeclarativeSubBlocksMetaclass, get_help_icon
+from .base import (
+    Block,
+    BoundBlock,
+    DeclarativeSubBlocksMetaclass,
+    get_error_json_data,
+    get_error_list_json_data,
+    get_help_icon,
+)
 
 __all__ = ["BaseStructBlock", "StructBlock", "StructValue"]
 
 
 class StructBlockValidationError(ValidationError):
```

Comment:
Sure - had the same thought myself! Planning to do that when I come to write the documentation, since that's the point at which this properly becomes a public-facing API.